tests/SchemaTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace tests\Libero\Schemas;
+
+use DOMDocument;
+use FluentDOM;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Finder;
+
+final class SchemaTest extends TestCase
+{
+    /**
+     * @dataProvider validFileProvider
+     */
+    public function testValid(DOMDocument $document, string $schema) : void
+    {
+        $result = $document->relaxNGValidate($schema);
+
+        $this->assertTrue($result);
+    }
+
+    /**
+     * @dataProvider invalidFileProvider
+     */
+    public function testInvalid(DOMDocument $document, string $schema) : void
+    {
+        $problems = [];
+        set_error_handler(
+            function (int $number, string $string) use (&$problems) : void {
+                $problems[] = $string;
+            }
+        );
+        $result = $document->relaxNGValidate($schema);
+        restore_error_handler();
+
+        $this->assertFalse($result);
+        $this->assertNotEmpty($problems);
```

Comment:
Would be useful to check the error messages, to make sure they're the right ones.